### ISSUE TITLE
Cannot read property 'ownerDocument' of null

### ISSUE THREAD

**Issue by lukeschunk** [focus-trap/focus-trap-react#44]
I'm getting this error thrown by the tabbable library which focus-trap-react depends on by way of focus-trap. At first I thought this might be a jest/enzyme problem but I'm also seeing it in non-test code. 

```
TypeError: Cannot read property 'ownerDocument' of null
    at tabbable (index.js:21)
    at updateTabbableNodes (index.js:267)
    at Object.activate (index.js:72)
    at FocusTrap.componentDidMount (focus-trap-react.js:55)
```

I also noticed that the line that's throwing this error is an older version than your most recent update of tabbable...perhaps a version update would help? I haven't tested that yet, but I would love any input you have. 

I've used this library before and had really great results with it, so thanks!

**Comment by pradel** [comment]
Same here, I upgraded from `4.0.1` to `6.0.0` and facing this issue

**Comment by pradel** [comment]
Also happening in `5.0.1`

**Comment by davidtheclark** [comment]
@lukeschunk can you please be more specific about the situation in which this behavior is happening? This does not happen universally — for example, I'm not seeing it in [the demos](https://davidtheclark.github.io/focus-trap-react/demo/) — so I think we need to identify what you and @pradel are doing that triggers the error.

**Comment by taasan** [comment]
I get the same error when trapping a SemanticUI react Modal

**Comment by davidtheclark** [comment]
I think the source of the error is https://github.com/davidtheclark/tabbable/blob/b8ceed435664fdf51b5f70729ab50c63e1877190/index.js#L142. Somehow in special cases we're arriving at that point and `node` is `null`. Probably adding an additional check would stop the error — but it would still be nice to understand *why* `node` might be `null` at that point.

There's a chain of libraries to update here — tabbable, focus-trap, then this one.

**Comment by lukeschunk** [comment]
I haven't had the chance to take a look yet, but I will when I can. 💥

**Comment by jimmaaay** [comment]
I found that if the child of FocusTrap isn't a HTML element then this error pops up.

E.g

Errors with `Cannot read property 'ownerDocument' of null`
```jsx
<FocusTrap>
  <Fragment>
    {/* ...content */}
  </Fragment>
</FocusTrap>
```

Works
```jsx
<FocusTrap>
  <div>
    {/* ...content */}
  </div>
</FocusTrap>
```

You can see an example [here](https://codesandbox.io/s/focus-trap-react-error-k0cph). Make sure its set to current module view. And checkout `will-error.js` & `works.js`

**Comment by lambert-velir** [comment]
I just ran in to this issue and @jimmaaay's fix works.  Is it possible to enable the use of `<Fragment>` also?

**Comment by ziven27** [comment]
```JSX
import FocusTrap from "focus-trap-react";
import { CSSTransition } from "react-transition-group";

<FocusTrap>
  <CSSTransition>
   <div>
    {/* ...content */}
  </div>
  </CSSTransition>
</FocusTrap>
```

I got the same error in this way.
is there anyway to define the ownerDocument on FocusTrap props?

**Comment by Cristy94** [comment]
I get same error, even if I wrap the FocusTrap contents in a `div`.

`Uncaught TypeError: Cannot read property 'ownerDocument' of null`

**Comment by rivajunior** [comment]
I think https://github.com/focus-trap/focus-trap-react/pull/139 fixed it

**Comment by stefcameron** [comment]
@rivajunior That would be awesome! I just published #139 in 8.1.1.
